file/file_util.cpp

```diff
@@ -25,7 +25,7 @@
 #include "file/file_util.h"
 #include "util/text/utf8.h"
 
-#if defined(__FreeBSD__) || defined(__APPLE__)
+#if !defined(__linux__) && !defined(__SYMBIAN32__)
```

Comment:
Elsewhere we have `#if !defined(__linux__) && !defined(_WIN32) && !defined(__QNX__)`, maybe that's best.  I think QNX is defined for Symbian.

Although, the define wouldn't be used on Windows so probably doesn't matter hugely.

-[Unknown]